### PR TITLE
Wt 95 setup notifications UI layout

### PR DESCRIPTION
**Pull request by k1mka**

### CODE REVIEW

File: lib/presentation/widgets/custom_button.dart [hunk 1]
```diff
@@ -24,7 +35,7 @@ class CustomButton extends StatelessWidget {
         padding: paddingAroundButton,
         child: Text(
           text,
-          style: CustomTheme.buttonSize,
+          style: TextStyle(fontSize: 16, color: textButtonColor),
```
Comment (zddarova) [inline lib/presentation/widgets/custom_button.dart]: Move to theme

File: lib/presentation/widgets/notification_widget.dart [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  static const paddingInsideButton = EdgeInsets.symmetric(horizontal: 24);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: heightWidget,
+      width: widthWidget,
+      decoration: BoxDecoration(
+        color: Colors.white,
+        borderRadius: const BorderRadius.only(
```
Comment (zddarova) [inline lib/presentation/widgets/notification_widget.dart]: .all or .circular?

File: lib/presentation/widgets/notification_widget.dart [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+          topLeft: widgetRadius,
+          topRight: widgetRadius,
+          bottomLeft: widgetRadius,
+          bottomRight: widgetRadius,
+        ),
+        boxShadow: [
+          BoxShadow(
+            color: Colors.grey.withOpacity(opacity),
```
Comment (zddarova) [inline lib/presentation/widgets/notification_widget.dart]: Move to theme? As shadow color

File: lib/presentation/widgets/sign_in_form.dart [hunk 4]
```diff
@@ -54,6 +54,8 @@ class _MyLogFormWidgetState extends State<SignInForm> {
             },
             text: LocaleKeys.sign_in.tr(),
             isEnabled: true,
+            buttonColor: CustomTheme.buttonDarkColor,
+            textButtonColor: Colors.white,
```
Comment (zddarova) [inline lib/presentation/widgets/sign_in_form.dart]: Move to theme

File: lib/presentation/widgets/sign_up_form.dart [hunk 5]
```diff
@@ -66,8 +66,9 @@ class _MyLogFormWidgetState extends State<SignUpForm> {
                 widget.onSignUpButtonPressed(_email.text, _pass.text);
               }
             },
-
             text: LocaleKeys.sign_up.tr(),
+            buttonColor: CustomTheme.buttonDarkColor,
+            textButtonColor: Colors.white,
```
Comment (zddarova) [inline lib/presentation/widgets/sign_up_form.dart]: Move to theme

File: lib/presentation/screens/notification_screen/notification_layout.dart [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:water_tracker/presentation/widgets/name_and_skip_widget.dart';
+import 'package:water_tracker/presentation/widgets/notification_widget.dart';
+
+class NotificationLayout extends StatelessWidget {
+  const NotificationLayout({Key? key}) : super(key: key);
+
+  static const paddingHorizontal = EdgeInsets.symmetric(horizontal: 24.0);
```
Comment (zddarova) [inline lib/presentation/screens/notification_screen/notification_layout.dart:8]: Many unused fields

File: lib/presentation/screens/goal_screen/goal_layout.dart [hunk 7]
```diff
@@ -90,11 +90,17 @@ class _GoalLayoutState extends State<GoalLayout> {
...
+                spacesBetween,
                 CustomButton(
-                  onPressed: () => context.read<GoalBloc>().add(SaveGoalEvent(goalsList)),
+                  onPressed: () {
+                    context.read<GoalBloc>().add(SaveGoalEvent(goalsList));
+                    Navigator.of(context).push(MaterialPageRoute(builder: (context) => const NotificationScreen()));
+                  },
                   text: LocaleKeys.next.tr(),
+                  buttonColor: CustomTheme.buttonDarkColor,
+                  textButtonColor: Colors.white,
```
Comment (zddarova) [inline lib/presentation/screens/goal_screen/goal_layout.dart]: Move to theme

File: lib/presentation/screens/goal_screen/goal_layout.dart [hunk 8]
```diff
@@ -16,35 +17,32 @@ import 'package:water_tracker/presentation/widgets/title_settings_widget.dart';
 class GoalLayout extends StatefulWidget {
   const GoalLayout({Key? key}) : super(key: key);
 
+  @override
+  State<GoalLayout> createState() => _GoalLayoutState();
+}
+
+class _GoalLayoutState extends State<GoalLayout> {
+  var goals = Goal.drinkWater;
+  List<Goal> goalsList = [];
```
Comment (zddarova) [inline lib/presentation/screens/goal_screen/goal_layout.dart]: Why is it here? And i don't see that it should work

**Review by zddarova — CHANGES_REQUESTED** [review]

File: lib/presentation/widgets/sign_in_form.dart (hunk 4 above)
Comment (k1mka) [reply lib/presentation/widgets/sign_in_form.dart]: it's already on theme

File: lib/presentation/widgets/sign_up_form.dart (hunk 5 above)
Comment (k1mka) [reply lib/presentation/widgets/sign_up_form.dart]: it's already on theme

File: lib/presentation/screens/notification_screen/notification_layout.dart (hunk 6 above)
Comment (k1mka) [reply lib/presentation/screens/notification_screen/notification_layout.dart:8]: there was not a single unused field

File: lib/presentation/screens/goal_screen/goal_layout.dart (hunk 7 above)
Comment (k1mka) [reply lib/presentation/screens/goal_screen/goal_layout.dart]: in theme yet

File: lib/presentation/screens/goal_screen/goal_layout.dart (hunk 8 above)
Comment (k1mka) [reply lib/presentation/screens/goal_screen/goal_layout.dart]: it works i checked

File: lib/presentation/screens/goal_screen/goal_layout.dart [hunk 9]
```diff
@@ -54,22 +51,24 @@ class _GoalLayoutState extends State<GoalLayout> {
...
+                  downFlex: downFlex,
                 ),
-                GoalLayout.spaces,
+                spaces,
                 Expanded(
                   child: GridView.count(
-                    mainAxisSpacing: GoalLayout.mainAxisSpacing.toDouble(),
-                    crossAxisSpacing: GoalLayout.crossAxisSpacing.toDouble(),
-                    crossAxisCount: GoalLayout.crossAxisCount,
+                    mainAxisSpacing: axisSpacing.toDouble(),
```
Comment (zddarova) [inline lib/presentation/screens/goal_screen/goal_layout.dart]: Make it double

**Review by zddarova — APPROVED** [review]

File: lib/presentation/screens/goal_screen/goal_layout.dart (hunk 8 above)
Comment (zddarova) [reply lib/presentation/screens/goal_screen/goal_layout.dart]: Goals variable not needed

**Review by zddarova — CHANGES_REQUESTED** [review]

**Review by zddarova — APPROVED** [review]

**Pull request merged**